src/components/accounts/create/status/AccountNeedsFunding.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { Translate } from 'react-localize-redux'
+import CopyIcon from '../../../svg/CopyIcon'
+import ClickToCopy from '../../../common/ClickToCopy'
+import Balance from '../../../common/Balance'
+import StyledContainer from './Style.css'
+
+const AccountNeedsFunding = ({
+    accountId,
+    fundingAddress,
```

Comment:
:nit: not a blocker, but it feels like `fundingAddress` and `accountId` are really the same thing, but there are two different `type`s of `accountId` -- `singleUse` vs. `nearName`, from a conceptual perspective (especially given `fundingAddress` really is a full-fledged account on-chain). From the engineering side, it's preferable to keep the concepts as close to reality as possible (reduces cognitive overhead and the numbers of ways we name the same logical _thing_), even if the UI tries to make the user feel like they are totally different things :)